src/packaging/packages.targets

```diff
@@ -60,13 +60,40 @@
             <ILCompilerSdkFiles Include="bootstrappercpp" />
             <ILCompilerSdkFiles Include="System.Private.CoreLib.Native" Condition="'$(OsEnvironment)'!='Windows_NT'" />
 
+            <!-- ILCompiler.SDK Cpp Codegen support files -->
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/env/common.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/env/etmdummy.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/env/gcenv.base.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/env/gcenv.object.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/env/gcenv.sync.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/gc.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/gcdesc.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/sample/gcenv.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/gcimpl.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/gcpriv.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/gcrecord.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/gcscan.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/handletable.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/handletable.inl" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/handletablepriv.h" />
+            <ILCompilerSdkCppCodgenFiles Include="Native/gc/objecthandle.h" />
+            <ILCompilerSdkCppCodgenCommonFiles Include="Native/Common/CPPCodegen/common.h" />
+            <ILCompilerSdkCppCodgenCommonFiles Include="Native/Common/CPPCodegen/platform.h" />
+
             <ILCompilerSdkFilesManaged Include="System.Private.CoreLib" />
+
             <ILCompilerSdkBinPlace Include="@(ILCompilerSdkFiles)">
                 <Text><![CDATA[        <file src="../lib/$(LibPrefix)%(Identity).$(StaticLibExt)" target="runtimes/$(NuPkgRid)/native/sdk/$(LibPrefix)%(Identity).$(StaticLibExt)" /> ]]></Text>
             </ILCompilerSdkBinPlace>
             <ILCompilerSdkBinPlace Include="@(ILCompilerSdkFilesManaged)">
                 <Text><![CDATA[        <file src="../%(Identity).dll" target="runtimes/$(NuPkgRid)/native/sdk/%(Identity).dll" /> ]]></Text>
             </ILCompilerSdkBinPlace>
+            <ILCompilerSdkBinPlace Include="@(ILCompilerSdkCppCodgenFiles)">
+                <Text><![CDATA[        <file src="$(SourceDir)%(Identity)" target="runtimes/$(NuPkgRid)/native/sdk/inc/%(Filename)%(Extension)" /> ]]></Text>
+            </ILCompilerSdkBinPlace>
```

Comment:
> native/ilc-sdk/x64_arm64/inc/

The SDK should be target specific only, it should not depend on platform of the host used for cross compilation. So this should be just `arm64`. 

Also, all header files in the CppCodeGen SDK should be platform agnostic, so the platform should not be in the path at all.